app/routes/surveys/components/SurveyEditor.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+
+import styles from './surveys.css';
+import React, { Component } from 'react';
+import { DetailNavigation, ListNavigation } from '../utils.js';
+import { Field } from 'redux-form';
+import Button from 'app/components/Button';
+import {
+  TextInput,
+  RadioButton,
+  SelectInput,
+  RadioButtonGroup,
+  DatePicker
+} from 'app/components/Form';
+import { createValidator, required } from 'app/utils/validation';
+import { reduxForm } from 'redux-form';
+import type { SurveyEntity } from 'app/reducers/surveys';
+import Content from 'app/components/Layout/Content';
+
+type Props = {
+  survey: SurveyEntity,
+  submitting: boolean,
```

Comment:
`type Props = FieldProps & {` from redux-form so you don't need handleSubmit and submitting